app/assets/javascripts/test/backend-snapshot-tests/tasks.e2e.js

```diff
@@ -87,35 +87,55 @@ test.serial("transferTask()", async t => {
   t.is(revertedTask.user && revertedTask.user.id, userId);
 });
 
-// Tests which require a working dataStore during tests and therefore don't work yet:
-//
-// test.serial("createTasks() and deleteTask()", async t => {
-//   const newTask = {
-//     boundingBox: null,
-//     dataSet: "confocal-multi_knossos",
-//     editPosition: [0, 0, 0],
-//     editRotation: [0, 0, 0],
-//     neededExperience: {
-//       domain: "abc",
-//       value: 1,
-//     },
-//     projectName: "Test_Project",
-//     scriptId: null,
-//     openInstances: 3,
-//     teamName: "570b9f4b2a7c0e3b008da6ec",
-//     taskTypeId: "570b9f4c2a7c0e4c008da6ee",
-//   };
-//   const createdTasks = await api.createTasks([newTask]);
-//   // console.log("createdTasks", createdTasks);
-//   t.snapshot(createdTasks, { id: "task-createTasks" });
-//
-//   await api.deleteTask(createdTasks[0].id);
-//   t.true(true);
-// });
-//
-// test.serial("requestTask()", async t => {
-//   const newTaskAnnotation = await api.requestTask();
-//   t.snapshot(newTaskAnnotation, { id: "task-requestTask" });
-// });
-//
+const newTask = {
+  boundingBox: null,
+  dataSet: "confocal-multi_knossos",
+  editPosition: [0, 0, 0],
```

Comment:
Can we use different values here? E.g., 

```
editPosition: [1, 2, 3],
editRotation: [4, 5, 6],
```

Zeros seem like a value, which the backend could also use as a fallback.